shared/desktop/renderer/index.js

```diff
@@ -155,7 +155,7 @@ function render(store, MainComponent) {
   }
   ReactDOM.render(
     <Root store={store}>
-      <div style={{display: 'flex', flex: 1}}>
+      <div style={{display: 'flex', flex: 1, '-webkit-app-region': 'drag'}}>
```

Comment:
we have a style for this: https://github.com/keybase/client/blob/master/shared/styles/index.desktop.js#L76

Reply:
Nice!